docs/icicle/colab-instructions.md

```diff
@@ -0,0 +1,131 @@
+# Run Icicle on Google Colab
```

Comment:
So this document wont render because it wasnt added to the side bar, i pused a commit adding it to the side bar.